common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -628,6 +635,10 @@ div.problem {
       height: ($baseline*2);
       vertical-align: middle;
       font-weight: 600;
+
+      @media print {
+        display: none;
```

Comment:
This only hides the buttons down the bottom of a problem to save/check/etc. Multiple choices still have their little round buttons, checkboxes are still present, and whatever the state the user presently has is what prints. The buttons to actually submit an answer and elicit a response from the server are all that I've stripped here.

For the purpose of having a hard copy of a page, be it for readability, the ability to scribble on the page, to just have a PDF record, or to journal/archive, the buttons down the bottom really don't add anything to the result, and can look misplaced to boot (especially a button that says "Hide Answers"). I honestly don't think that much if anything will be lost by hiding these buttons.